Formula/vvdec.rb

```diff
@@ -0,0 +1,27 @@
+class Vvdec < Formula
+  desc "Fraunhofer Versatile Video Decoder"
+  homepage "https://github.com/fraunhoferhhi/vvdec"
+  url "https://github.com/fraunhoferhhi/vvdec/archive/refs/tags/v1.6.1.tar.gz"
+  sha256 "72de0cbcd24285e6f66209be9270f8f0c897d24e586b3876c6a7bb5691375c48"
+  license "BSD-3-Clause-Clear"
+  head "https://github.com/fraunhoferhhi/vvdec.git", branch: "master"
+
+  depends_on "cmake" => :build
+
+  resource("test_video") do
+    url "https://github.com/pthariensflame/homebrew-assorted/raw/master/test-fixtures/test.vvc"
+    sha256 "753261009b6472758cde0dee2c004ff712823b43e62ec3734f0f46380bec8e46"
+  end
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DVVDEC_INSTALL_VVDECAPP=1"
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build",
                    "-DBUILD_SHARED_LIBS=1",
                    "-DVVDEC_INSTALL_VVDECAPP=1",
                    *std_cmake_args
```
Let's build shared instead of static libraries. Also, always pass `std_cmake_args` last.